iOS/Flows/Conversation/Cells/ThreadMessageCell.swift

```diff
@@ -116,3 +126,41 @@ class ThreadMessageCell: UICollectionViewCell {
         self.setNeedsLayout()
     }
 }
+
+// MARK: - UIContextMenuInteractionDelegate
+
+extension ThreadMessageCell: UIContextMenuInteractionDelegate {
```

Comment:
Basically the same thing as the message cell except that there's no "open thread" action.